src/WebJobs.Script/OutOfProc/Http/DefaultHttpInvokerService.cs

```diff
@@ -3,23 +3,90 @@
 
 using System;
 using System.Net.Http;
+using System.Net.Http.Formatting;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
 using Microsoft.Azure.WebJobs.Script.Description;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
 
-namespace Microsoft.Azure.WebJobs.Script.OutOfProc
+namespace Microsoft.Azure.WebJobs.Script.OutOfProc.Http
 {
     public class DefaultHttpInvokerService : IHttpInvokerService
     {
         private readonly HttpClient _httpClient;
+        private readonly string _httpInvokerBaseUrl;
 
         public DefaultHttpInvokerService(HttpClient httpClient)
         {
             _httpClient = httpClient;
+            // TODO: user agent ok?
+            _httpClient.DefaultRequestHeaders.UserAgent.ParseAdd(HttpInvokerConstants.UserAgentHeaderValue);
```

Comment:
For a http triggered function, do we preserve the user agent sent from the client?

Reply:
For simple http triggers, i will update the code to forward the request to the invoker. For functions that use multiple inputs/ouputs, httpTrigger will be part of the input data